riot/lol/match.go

```diff
@@ -14,6 +16,39 @@ type MatchClient struct {
 	c *internal.Client
 }
 
+// MatchListOptions providing additional options for List
+type MatchListOptions struct {
+	// Set of champion IDs for filtering the matchlist.
+	Champion []int
+	// Set of queue IDs for filtering the matchlist.
+	Queue []int
+	// The begin time to use for filtering matchlist
+	BeginTime time.Time
+	// The end time to use for filtering matchlist
+	EndTime time.Time
+}
+
+func (mo *MatchListOptions) buildParam() string {
+	var param string
+	if len(mo.Champion) != 0 {
+		for _, champion := range mo.Champion {
+			param += "&champion=" + strconv.Itoa(champion)
+		}
+	}
+	if len(mo.Queue) != 0 {
+		for _, queue := range mo.Queue {
+			param += "&queue=" + strconv.Itoa(queue)
+		}
+	}
+	if mo.BeginTime.Unix() != -62135596800 {
+		param += "&beginTime=" + strconv.FormatInt(mo.BeginTime.UnixNano()/int64(time.Millisecond), 10)
```

Comment:
```suggestion
	if !mo.BeginTime.IsZero() {
		param += "&beginTime=" + strconv.Itoa(int(mo.BeginTime.UnixNano() / 1000 / 1000))
```
I think we should use the more straightforward methods `strconv.Itoa` and `time.Time.IsZero` for this.
Same goes for the handling of `EndTime` below.

Reply:
Thanks for the good advice. 
fixed!